gp2gp-translator/src/test/java/uk/nhs/adaptors/pss/translator/mapper/diagnosticreport/SpecimenMapperTest.java

```diff
@@ -45,57 +56,124 @@ public class SpecimenMapperTest {
     private static final String DR_ID = "DIAGNOSTIC_REPORT_ID";
     private static final DiagnosticReport DIAGNOSTIC_REPORT_WITH_SPECIMEN = generateDiagnosticReportWithSpecimenReference();
     private static final DiagnosticReport DIAGNOSTIC_REPORT_WITHOUT_SPECIMEN = generateDiagnosticReportWithNoSpecimenReference();
+    private static final String SPECIMEN_META_PROFILE = "Specimen-1";
+    private static final Meta META_WITH_SECURITY_ADDED = MetaFactory.getMetaFor(META_WITH_SECURITY, SPECIMEN_META_PROFILE);
+    private static final CV NOPAT_CV = TestUtility.createCv(
+        "NOPAT",
+        "http://hl7.org/fhir/v3/ActCode",
+        "no disclosure to patient, family or caregivers without attending provider's authorization");
 
     private static final String NARRATIVE_STATEMENT_ID = "9326C01E-488B-4EDF-B9C9-529E69EE0361";
 
     @Mock
     private DateTimeMapper dateTimeMapper;
 
+    @Mock
+    private ConfidentialityService confidentialityService;
+
     @InjectMocks
     private SpecimenMapper specimenMapper;
 
     @Test
-    public void testSpecimenIsMapped() {
+    public void testHandlingSpecimenNoPatCompoundStatement() {
```

Comment:
I think we can be clearer with the test names 😄 maybe something like `testHandlingSpecimenNoPatCompoundStatementExpectMetaSecurityPresent` or something? What's your thoughts!

Reply:
yes, it will be fixed